lib/ping_pong/events/life_start.js

```diff
@@ -53,7 +54,7 @@ const onLifeStart = async (instance = {}) => {
   }
 
   // Handle saved pongs
-  const pongOrders = Object.keys(activePongs).map(price => (
+  const pongOrders = getPings(activePongs).map(price => (
```

Comment:
not getPongs?

Reply:
according to the previous logic, it should extract values ​​from the first column (previously it was a key)